src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -48,13 +47,15 @@ function range<T>(length: number, valueFunction: (index: number) => T): T[] {
   return valuesArray;
 }
 
+/** InjectionToken for datepicker that can be used to override default locale code. */
+export const MAT_DATE_LOCALE = new InjectionToken<string>('MAT_DATE_LOCALE');
```

Comment:
can we move this to `DateAdapter` since we expect all adapters to use it not just `NativeDateAdapter`?